proc_scripts/phasing.py

```diff
@@ -201,12 +201,12 @@ def hermitian_function_test(s, down_from_max=0.5, rel_shift=1.5,shift_points=120
     data_for_peak /= max_val
     pairs = data_for_peak.contiguous(lambda x: abs(x) >
             down_from_max)
-    peak_location = pairs[0,:] #gives the longest pair
-    peak_center = peak_location.mean()
+    peak_bounds = pairs[0,:] #gives the longest pair
+    peak_center = data_for_peak['t2':(0.1e-3,None)].argmax('t2').item()
```

Comment:
this makes sense seeing it but definitely was having trouble visualizing it as one line.